docs/user-guide/petset.md

```diff
@@ -143,16 +145,52 @@ Lets verify this assertion with a simple test.
 $ kubectl get svc
 NAME          CLUSTER-IP     EXTERNAL-IP       PORT(S)   AGE
 nginx         None           <none>            80/TCP    12m
+...
+```
+
+First, note that the PetSet gives us a stable hostname:
```

Comment:
First, the Pet Set provides a stable hostname:

Question though: It looks like you're *setting* the hostname in this sh script?